doc/tutorials/app/table_of_content_app.markdown

```diff
@@ -1,6 +1,9 @@
 Application utils (highgui, imgcodecs, videoio modules) {#tutorial_table_of_content_app}
 =======================================================
 
+@prev_tutorial{tutorial_table_of_content_imgproc}
+@next_tutorial{tutorial_table_of_content_calib3d}
+
```

Comment:
I'm against cross-module prev-next links in documentation by two reasons at least:
- The "next" link jumps to dramatically different scope. The next page is from some other context.
- 5.x and 4.x has slightly different set of modules. I have to tune links each 4.x-5.x merge.